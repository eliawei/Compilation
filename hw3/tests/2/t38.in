void main() {
    callee();
}

void callee() {
    print("not ok");
}