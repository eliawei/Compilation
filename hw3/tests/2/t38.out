line 2: function callee is not defined
